client/my-sites/email/paths.ts

```diff
@@ -150,7 +150,9 @@ export const getEmailManagementPath: EmailPathUtilityFunction = (
 	urlParameters
 ) => {
 	if ( isUnderDomainManagementAll( relativeTo ) ) {
-		return getPath( siteName, domainName, null, relativeTo, urlParameters );
+		return `${ domainsManagementPrefix }/${ siteName }/${ domainName }${ buildQueryString(
+			urlParameters
+		) }`;
 	}
```

Comment:
The similar approach as we did here:
https://github.com/Automattic/wp-calypso/blob/1f1cfcbeea6635788a9b1abcf755ca0555cb7c11/client/my-sites/email/paths.ts#L79-L83